### PR TITLE
Increase coverage of sw and manifest routes

### LINKED ISSUE #1967: Increase test coverage of sw and manifest routes
**Is your feature request related to a problem? Please describe.**
As can be seen in the image below, the two routes for sw.js and manifest.json lack test coverage. This is primarily due to how the routes dont work, and throw a 500, when tested. They are covered by cypress tests, but thats not accounted for in test coverage 

image

**Describe the solution you'd like**
Write tests to cover them. Maybe use `jest-express` to cover them without requiring the real file be to available?

**Testing notes**
[Tester to complete]

Dev insight: Unit tests

### PR DESCRIPTION
**Pull request by Bopchy**
Resolves #1967 

**Overall change:** _Add unit tests for the sw and manifest routes._

**Code changes:**

- _Add unit tests._
---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/server/index.test.jsx [hunk 1]
```diff
@@ -10,8 +10,12 @@ import { localBaseUrl } from '../testHelpers/config';
 // mimic the logic in `src/index.js` which imports the `server/index.jsx`
 dotenv.config({ path: './envConfig/local.env' });
 
+const getPath = require('path');
```
Comment (ghost) [inline src/server/index.test.jsx]:
```suggestion
const path = require('path');
```

File: src/server/index.test.jsx [hunk 2]
```diff
@@ -398,29 +402,35 @@ describe('Server', () => {
...
-          data: { some: 'data' },
-          service: 'someService',
-          status: 404,
-        };
+  describe('Service workers', () => {
+    // sends SW file for paths matching regex
+    it('should serve a file for existing service workers', async () => {
+      await makeRequest('/news/articles/sw.js');
+      expect(spy.mock.calls[0][0]).toEqual(
+        getPath.join(__dirname, '/public/sw.js'),
```
Comment (ghost) [inline src/server/index.test.jsx]:
```suggestion
        path.join(__dirname, '/public/sw.js'),
```

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/server/index.test.jsx [hunk 3]
```diff
@@ -10,8 +10,12 @@ import { localBaseUrl } from '../testHelpers/config';
 // mimic the logic in `src/index.js` which imports the `server/index.jsx`
 dotenv.config({ path: './envConfig/local.env' });
 
+const getPath = require('path');
+const express = require('express');
 const server = require('./index').default;
 
+const spy = jest.spyOn(express.response, 'sendFile');
```
Comment (dr3) [inline src/server/index.test.jsx]:
```suggestion
const sendFileSpy = jest.spyOn(express.response, 'sendFile');
```

File: src/server/index.test.jsx (hunk 1 above)
Comment (Bopchy) [reply src/server/index.test.jsx]: Made this change though - https://github.com/bbc/simorgh/blob/20fe38d99939d085fea6abf6bd1c2ef5e4dd6d31/src/server/index.test.jsx#L68 to deal with the eslint error of `path` already being defined in the upper scope

File: src/server/index.test.jsx [hunk 4]
```diff
@@ -398,29 +402,35 @@ describe('Server', () => {
...
-    describe('Services not on allowlist', () => {
-      beforeEach(() => {
-        const notFoundDataResponse = {
-          isAmp: false,
-          data: { some: 'data' },
-          service: 'someService',
-          status: 404,
-        };
+  describe('Service workers', () => {
+    // sends SW file for paths matching regex
```
Comment (ghost) [inline src/server/index.test.jsx]: Is this comment necessary?

Comment (Bopchy) [reply src/server/index.test.jsx]: I am ok with removing the comments

Comment (ghost) [reply src/server/index.test.jsx]: It talks about `matching regex` but i see no regex within the test.

Comment (Bopchy) [reply src/server/index.test.jsx]: It referred to the regex used here - https://github.com/bbc/simorgh/blob/e8f2d6ad5247be459e425c9a0216f233f2300186/src/server/index.jsx#L151. But I think the test description and the test are easy to understand without the comment, so I've removed it and the one on the manifest test

**Review by j-pendlebury — APPROVED** [review]
LGTM 👍

**Review by ghost — APPROVED** [review]

**Pull request merged**